types/ioredis/index.d.ts

```diff
@@ -344,8 +345,7 @@ declare namespace IORedis {
         type(key: string, callback: (err: Error, res: string) => void): void;
         type(key: string): Promise<string>;
 
-        multi(callback: (err: Error, res: string) => void): void;
-        multi(): Promise<string>;
+        multi(commands?: string[][], options?: MultiOptions): Pipeline;
```

Comment:
Looks good, except need to add one override function of `multi(options?: MultiOptions): Pipeline;`, because I see a code example of `redis.multi({ pipeline: false });`

Reply:
How about `multi(options: { pipeline: false }): Promise<string>`? I tried it and it doesn't take a callback with `pipeline: false` (and no array commands either) so this could be the only overload needed.

Reply:
OK